test/ui/test_spec.js

```diff
@@ -168,6 +168,37 @@ describe('UI tests', function() {
         .then(done);
     });
 
+    it('Settings page defaults to light theme', function(done) {
+      driver.findElement(By.id('accountDropdownButton')).click()
+        .then(() => screenshotAndCompare('settingsMenu.png', 'settingsMenu', null))
+        .then(() => driver.findElement(By.id('settingsButton')).click())
+        // After clicking on the Settings menu item, wait for the dialog.
+        .then(() => driver.wait(until.elementLocated(By.id('lightThemeRadioOption'))))
+        // Wait for the fade-in to complete.
+        .then(() => driver.sleep(1000))
```

Comment:
do we really need this? is the test flaky without it? if so, is there maybe a better way?